features/step_definitions/submit_review_failure.rb

```diff
@@ -0,0 +1,71 @@
+Given(/^I am on the new review page$/) do
```

Comment:
Not this one specifically, but a lot of the steps in this file are essentially duplicated in the `submit_review_success.rb` file. Could we refactor those steps into their own `submit_review.rb` file so that we have one source of steps for submitting reviews?

Then the `submit_review_failure.rb` and `submit_review_success.rb` files can have failure/success-specific steps, respectively.

Reply:
All of the steps are the same until the redirect step at the end. Success should redirect to prof page while failure should stay on the same page. How do I do that in only one step file?